Resources/Prototypes/Entities/Mobs/Customization/Markings/reptilian.yml

```diff
@@ -271,3 +271,12 @@
   sprites:
   - sprite: Mobs/Customization/reptilian_parts.rsi
     state: horns_kobold_ears
+    
+- type: marking
+  id: LizardHornsFloppyKoboldEars
+  bodyPart: HeadSide
+  markingCategory: HeadSide
```

Comment:
Was this intended to be HeadSide? Does making it HeadSide allow, for example, another HeadTop item to be selected in addition to this marking?

Reply:
Yeah, a couple people were asking me to have it be HeadSide so that they could use it with the horns.